metadata.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+# This file populates the Overview on Charmhub.
+# See https://juju.is/docs/sdk/metadata-reference for a checklist and guidance.
+
+name: superset-k8s
+
+display-name: Superset Operator
+
+summary: Superset operator
+
+description: |
+  Superset is fast, lightweight, intuitive, and loaded with options that make it easy for users of all skill sets to explore and visualize their data.
```

Comment:
But what is Superset? I think a start like `Superset is a BI tool that is fast, ...` would be more descriptive. Ignore this if you want though.

Reply:
Good point. Updated.